packages/mermaid/src/defaultConfig.ts

```diff
@@ -8,19 +8,27 @@ import { MermaidConfig } from './config.type';
  *
  * These are the default options which can be overridden with the initialization call like so:
  *
- * **Example 1:**<pre> mermaid.initialize({ flowchart:{ htmlLabels: false } }); </pre>
+ * **Example 1:**
  *
- * **Example 2:**<pre> <script> var config = { startOnLoad:true, flowchart:{ useMaxWidth:true,
- * htmlLabels:true, curve:'cardinal', },
+ * ```js
+ * mermaid.initialize({ flowchart:{ htmlLabels: false } });
+ * ```
  *
- *     securityLevel:'loose',
+ * **Example 2:**
  *
- * }; mermaid.initialize(config); </script> </pre>
+ * ```html
+ * <script>
+ * var config = {
+ *   startOnLoad:true,
+ *   flowchart:{ useMaxWidth:true, htmlLabels:true, curve:'cardinal'},
+ *   securityLevel:'loose',
+ * };
+ * mermaid.initialize(config);
+ * </script>
+ * ```
```

Comment:
It looks like these docs are out-of-sync with `docs/Setup.md`, since the `postbuild` action was removed when moving to the mono-repo.

https://github.com/mermaid-js/mermaid/blob/f2279d48d84921d64463bac4cefc14c96ef6dd58/package.json#L36

I had a quick try to fix it, but it looks like [`documentation`](https://github.com/documentationjs/documentation) has some issue with the current version of `vite` (maybe because `pnpm` makes them share dependencies?) so it fails to run.

However, I reckon we should replace `documentation` anyway, since it's barely maintained and the latest version has a bug that breaks how we're using it anyway.

Instead, we can replace it with TypeDoc using [typedoc-plugin-markdown](https://www.npmjs.com/package/typedoc-plugin-markdown) to output a markdown file.